### PR TITLE
Added information about caching

### PR DESCRIPTION
**Pull request by michaelstrange**
Added info about caching.. :-)

### CODE REVIEW

**Comment by BYK** [comment]
Hey, thanks for the patch. Can you squash your patches?

**Comment by BYK** [comment]
>your patches?

I mean commit.s

File: README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 [GitHub Badge][ghb] is a simple embeddable badge showing your [GitHub][github] stats
 like the number of public repositories, number of followers, favorite languages etc.
 
+## Please note:
+We do rely on heavy caching so give it about 24 to 48 hours to pick up these changes
```
Comment (BYK) [inline README.md]: these -> any.

Also please add a period at the end of the sentence.

**Comment by michaelstrange** [comment]
Sorry.. That's how things go when you're too busy. I will fix it.

**Comment by michaelstrange** [comment]
sry.. i'll create e new one

**Pull request closed without merging**